client/lib/application/src/version-detail/version-detail.js

```diff
@@ -0,0 +1,48 @@
+import BaseView from 'common/src/base-view';
+import Template from './version-detail.hbs';
+import Flux from 'application/src/flux';
+import FetchResult from 'common/src/fetch-result';
+import ErrorTpl from 'common/src/error.hbs';
+import Placeholder from './placeholder.hbs';
+import Markdown from 'common/src/markdown';
+
+import 'common/asset/scss/application/application-detail.scss';
```

Comment:
as currently has no differences with application-detail, i decided to not replicate code.